website/blog/2020-07-30-7.11.0.md

```diff
@@ -145,11 +151,11 @@ try {
 ```
 Note that currently only `unknown` and `any` are allowed in catch binding type annotations. `@babel/parser` does not enforce this check because alias `type ANY = any` or  intersection `any | unknown` are not evaluated at compile-time.
 
-From Babel 7.11 you can use these new features without any config changes. For more information, please checkout [TypeScript 4.0 Announcement](https://devblogs.microsoft.com/typescript/announcing-typescript-4-0-beta).
+Starting from Babel 7.11, you can use these new features without any config changes. For more information, please checkout [TypeScript 4.0 Announcement](https://devblogs.microsoft.com/typescript/announcing-typescript-4-0-beta).
 
 ## Decimal Literal parser support ([#11640](https://github.com/babel/babel/pull/11640))
 
-[Decimal Literal Proposal](https://github.com/tc39/proposal-decimal) (Stage 1) provides notation for decimals, analogous to BigInt and integers.
+The [Decimal Literal Proposal](https://github.com/tc39/proposal-decimal) (Stage 1) provides notation for infinite-precision decimal numbers, analogous to BigInt and integers.
```

Comment:
The proposal champion has not decided whether BigDecimal or Decimal should be pursuit: https://github.com/tc39/proposal-decimal#option-decimal128 `Decimal128` has finite precision.